jsonapi/jsonapi_utility.go

```diff
@@ -141,33 +141,33 @@ type Conflict interface {
 // JSONErrorResponse auto maps the provided error to the correct response type
 // If all else fails, InternalServerError is returned
 func JSONErrorResponse(obj interface{}, err error) error {
-	x := obj.(InternalServerError)
-	c := obj.(context.Context)
-
-	jsonErr, status := ErrorToJSONAPIErrors(c, err)
+	ctx := obj.(context.Context)
```

Comment:
Please add a conversion check here, just to be safe.

Reply:
ok, I'll do that

Reply:
actually, replacing the `obj interface{}` parameter with `ctx context.Context` is even more straightforward and makes more sense ;)